internal/pkg/deploy/cloudformation/stack/testdata/workloads/rdws-test.stack.yml

```diff
@@ -153,6 +153,7 @@ Resources:
         EgressConfiguration:
           EgressType: VPC
           VpcConnectorArn: !Ref VpcConnector
+      AutoScalingConfigurationArn: 'arn:aws:apprunner:us-east-1:123456789012:autoscalingconfiguration/high-availability/3'    
```

Comment:
Do we need to accept an entire ARN? can we not generate the ARN ourselves so that users don't have to write the account ID and region and instead they can pass just `highavailability/3`?

In cloudformation, we can do something like:
https://github.com/aws/copilot-cli/blob/5781c535bc78e5e3a544b855d930256562e52b63/internal/pkg/deploy/cloudformation/stack/testdata/pipeline/cc_template.yaml#L97